### DISCUSSION TITLE
Sensor setup: Installer version mismatch error

### DISCUSSION THREAD

**Discussion by kiaraho** [Security-Onion-Solutions/securityonion#2414] [Unsupported Versions] [answered]
Adding a new Sensor to an existing grid using `so-setup-network` on Ubuntu 18. Ran into the error below after entering the `soremote` password.
```
Now try logging into the machine, with:   "ssh 'soremote@redacted'"
and check to make sure that only the key(s) you wanted were added.

2020-12-22T00:09:10Z | I | Installer version mismatch, downloading correct version from manager
scp: /opt/so/repo/2.3.10.tar.gz: No such file or directory
Could not download 2.3.10.tar.gz from manager, please check your network settings and verify the file /opt/so/repo/2.3.10.tar.gz exists on the manager.
```
The `/opt/so/repo/` directory does not exist on the Manager. Is there a command should I run to populate this directory? Manager is running version `2.3.10`.

**Comment by TOoSmOotH** [comment, marked as answer]
You are running a 2.3.20 install script against a 2.3.10 manager. You need to soup your manager. We will be releasing 2.3.21 in a few hours so it might be worth waiting for that to happen.

**Reply by kiaraho** [reply]
@TOoSmOotH, thanks for the quick response. I'll give it 24hrs before trying again. I'm assuming I also need to `soup` the SearchNode after updating the Manager?

**Reply by TOoSmOotH** [reply]
2.3.21 is up there. No soup only runs on the manager. The manager will handle the rest.

**Comment by mendo1024** [comment]
I had a similar issue and I solved it by cloning and using the identical version as on the manager:

`git clone --depth 1 --branch '2.3.10' https://github.com/Security-Onion-Solutions/securityonion`

**Reply by ghost** [reply]
> I had a similar issue and I solved it by cloning and using the identical version as on the manager:
> 
> `git clone --depth 1 --branch '2.3.10' https://github.com/Security-Onion-Solutions/securityonion`

This was a huge help. I used it to install 2.3.40 today. Thank you!!!
